### ISSUE TITLE
"View as PDF" function not working

### ISSUE THREAD

**Issue by thnhub** [docker/compose#8952]
I have successfully installed the DMOJ website on ubuntu 20.04. However,  the "View as PDF" function not working. I see the "View as PDF" button on-page, but it occurs an error when I click on it. 
This is the error:

```
"SIGSEGV: invalid_state
corrupt page /problem/aplusb/pdf
site: fatal signal: Segmentation fault
site died (signal 500, exit -11)
panic: go back"
```

How can I resolve that error, please guide me? Thank you!

**Comment by ndeloof** [comment]
How does this relate to docker compose?

**Comment by thnhub** [comment]
> How does this relate to docker compose?

Docker compose process is normal, but I get the following error when I click on the "View as PDF" button:
```
selenium.common.exceptions.WebDriverException: Message: Service /usr/bin/google-chrome unexpectedly exited. Status code was: 1
```
As DMOJ docs this error relates to Selenium and ChromeDriver, and I tried to search for this on many websites but I still cannot fix it. Could you guide me, or point me to a forum where I can ask this question? Thank you very much!

**Comment by ndeloof** [comment]
What I mean is: can you build and run this Docker image using plain `docker build` and `docker run`?
There's nothing here that describe a compose issue we could help you to fix.

**Comment by thnhub** [comment]
> What I mean is: can you build and run this Docker image using plain `docker build` and `docker run`? There's nothing here that describe a compose issue we could help you to fix.

Yes, I can. I have built successfully, and now running the DMOJ site on my PC. During the composing by `docker build` I saw some warnings, but all was well. However, I didn't get those warnings back. The DMOJ site is working very well, except cannot render a PDF file by "View as PDF" function that I know. It is possible that this issue is not coming from composing.
Thank you @ndeloof!
